src/test/java/hudson/plugins/ec2/AmazonEC2CloudUnitTest.java

```diff
@@ -36,6 +39,10 @@
 * Unit tests related to {@link AmazonEC2Cloud}, but do not require a Jenkins instance.
 */
 public class AmazonEC2CloudUnitTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
```

Comment:
Why do we need jenkinsrule?

Reply:
_AmazonEC2Cloud_ now relies on a call to SystemCredentialsProvider.getInstance() which is not available without a jenkinsrule. Is there another way to solve this?

Reply:
AFAIK, no. If it is needed thats fine then